pkg/kv/kvserver/kvflowcontrol/rac2/range_controller.go

```diff
@@ -91,6 +92,13 @@ type RangeController interface {
 	InspectRaftMuLocked(ctx context.Context) kvflowinspectpb.Handle
 }
 
+// RaftInterface implements methods needed by RangeController.
+//
+// Locking: as noted in replicaSendStream, replicaSendStream.mu is ordered
+// after Replica.mu. So avoid acquiring Replica.mu in the implementation of
+// RaftInterface (to prevent deadlock), or very carefully reason through the
+// consequences.
```

Comment:
The `RaftInterface` only interacts with `RawNode`, and no send stream stuff, so `mu` locking inside this interface won't lead to any deadlocks and seems irrelevant. The ordering with the send stream is the `RangeController`'s concern (specifically the "send pings" method's)?

It's not like we don't want to lock `mu` here, it's that we are required to lock it (for reads or writes depending on the method) because that is the `RawNode` contract, and this type is a thin wrapper.

For the same reason, `FollowerStateRaftMuLocked` will change too.